nds-org/ndslabs
# fix: jquery version mismatch caused mobile navbar to fail

## Problem
A version mismatch exists between Bootstrap 3.4.1 and jQuery 3.5.0 (see https://github.com/twbs/bootstrap/issues/30644)

## Approach
Roll jQuery forward to `^3.6.0`, which currently installs jQuery `3.6.1`

## How to Test
1. Checkout and run this branch locally
2. Navigate to the Workbench Landing page
3. Open the developer console and switch to mobile view (Elements, at the top-left - icon looks like a phone)
4. Click on the navbar hamburger menu to expand it
    * You should see the navbar now appears (before this silently threw an error in the console)
